apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDevelop.md

```diff
@@ -1,184 +1,56 @@
+```shell
+git clone https://github.com/kenotron/create-react-app-uifabric.git my-app
+cd my-app
+yarn
+yarn start
 ```
 
-Open `index.html` file.
-
-Fabric React components provide support for Left-to-Right (LTR) and Right-to-Left (RTL) rendering out of the box.
+## Gatsby.js
 
-You can specify the default direction in your `index.html`. Add the `dir` attribute with the value ltr to the `<html>` element:
-
-```html
-<html lang="en" dir="ltr"></html>
-```
+To create blazing fast React websites and apps using UI Fabric and Gatsby.js, use the following starter kit:
 
-Load Office UI Fabric styles by linking to the Office UI Fabric CDN. Add the following to the `<head>` element:
-
-```html
-<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.6.1/css/fabric.min.css" />
-```
-
-Save the file.
-
-### Step 5: Preview button
-
-Switch to the console and type the following to preview your app in the browser:
-
-```bash
-npm start
+```shell
+npm install -g gatsby-cli
+gatsby new gatsby-site kenotron/gatsby-starter-uifabric
+cd gatsby-site
+gatsby develop
 ```
 
-You should see the `PrimaryButton` component in your app.
-
-### Step 6: Using UI Fabric Icons
-
-If you are using Fabric React components that have icons, you can make all icons available by calling the `initializeIcons` function from the `@uifabric/icons` package.
-
-1. Install the `@uifabric/icons` npm package.
-
-   ```bash
-   npm install @uifabric/icons
-   ```
-
-2. Import `initializeIcons` function and then call it within your App.
-
-   ```tsx
-   import { initializeIcons } from '@uifabric/icons';
-
-   // Register icons and pull the fonts from the default SharePoint CDN:
-   initializeIcons();
-
-   // ...or, register icons and pull the fonts from your own CDN:
-   initializeIcons('https://my.cdn.com/path/to/icons/');
-   ```
-
-This will make ALL icons in the collection available, but will download them on demand when referenced using the `Icon` component.
-
-### Next steps
-
-If you are using Fabric React components that have icons, you can make all icons available by calling the `initializeIcons` function from the `@uifabric/icons` package.
-
-1. Install the `@uifabric/icons` npm package.
-
-   ```bash
-   npm install @uifabric/icons
-   ```
-
-2. Import `initializeIcons` function and then call it within your App.
+You can even read about how to deploy this to the cloud in one click here:
 
-   ```tsx
-   import { initializeIcons } from '@uifabric/icons';
+https://github.com/kenotron/gatsby-starter-uifabric#-deploy
 
-   // Register icons and pull the fonts from the default SharePoint CDN:
-   initializeIcons();
+## Next Steps
 
-   // ...or, register icons and pull the fonts from your own CDN:
-   initializeIcons('https://my.cdn.com/path/to/icons/');
-   ```
+If you are interested in learning more about how to use UI Fabric, here are few resources:
```

Comment:
This should probably be a paragraph, not a list, and lead with the controls. Something like:

`Now you're ready to [explore Fabric controls](#/controls/web) and use them in your app. To learn more about developing and testing Fabric React apps, see the day 2 and bonus sections of this [frontend bootcamp material](https://microsoft.github.io/frontend-bootcamp/).`